Source/Core/DolphinQt/Config/Mapping/MappingWidget.cpp

```diff
@@ -173,19 +173,19 @@ QGroupBox* MappingWidget::CreateGroupBox(const QString& name, ControllerEmu::Con
     form_layout->insertRow(2, mouse_button);
 
     using ControllerEmu::Cursor;
-    connect(mouse_button, &QCheckBox::clicked, [this, group = static_cast<Cursor*>(group)] {
+    connect(mouse_button, &QCheckBox::clicked, [this, grp = static_cast<Cursor*>(group)] {
       std::string default_device = g_controller_interface.GetDefaultDeviceString() + ":";
       const std::string controller_device = GetController()->GetDefaultDevice().ToString() + ":";
       if (default_device == controller_device)
       {
         default_device.clear();
       }
-      group->SetControlExpression(0, fmt::format("`{}Cursor Y-`", default_device));
-      group->SetControlExpression(1, fmt::format("`{}Cursor Y+`", default_device));
-      group->SetControlExpression(2, fmt::format("`{}Cursor X-`", default_device));
-      group->SetControlExpression(3, fmt::format("`{}Cursor X+`", default_device));
+      grp->SetControlExpression(0, fmt::format("`{}Cursor Y-`", default_device));
+      grp->SetControlExpression(1, fmt::format("`{}Cursor Y+`", default_device));
+      grp->SetControlExpression(2, fmt::format("`{}Cursor X-`", default_device));
+      grp->SetControlExpression(3, fmt::format("`{}Cursor X+`", default_device));
```

Comment:
My only gripe is this seems like a weird thing to rename, but otherwise it looks fine.

Reply:
The name shadowed a variable of the same name in the upper scope. I mentioned in the commit message that it fixed a warning. ;)